sections/related_works.tex

```diff
@@ -1,33 +1,41 @@
 \section{Related Work}\label{sec:related_work}
-
 \citet{mimno2008topic} describe two general topic model patterns, called ''upstream`` and ''downstream``.
 These patterns describe how metadata is incorporated in topic models.
 In the downstream pattern, metadata is incorporated in the model by having the topic model generate both the words and the metadata simultaneously.
 Here, each topic has a distribution over words, as well as a distribution over metadata values.
 In the upstream pattern, the topic model is conditioned on metadata elements by representing document-topic distributions as element specific distributions.
 This way the model learns assignments of the words in each document to the metadata.
+The models, we create, follow the upstream pattern.
 
 \citet{author_topic_2012} have developed an \gls{lda} model called the Author-Topic model, which incorporates authorship information in the \gls{lda} model.
 Specifically, each document has a vector of authors, where for each word an author is drawn from this vector.
 The author is then used in combination with an author-topic distribution to draw a specific topic that this author writes about.
 The words from the topic-word distribution can be generated based on this topic.
 The purpose of using authorship information in this way, is to show patterns in which topics an author usually writes about, and be able to explore how related authors are in what they write about.
 \citeauthor{author_topic_2012} also show that the combination of authorship and \gls{lda} yield more coherent topics.
+However, \citet{author_topic_2012} is applied to scientific article datasets, where the authors usually write about the same subject.
+We apply the Author-Topic model on a news article dataset to see whether similar performance can be obtained.
 
 \citet{MetaLDA2017} present a model, called MetaLDA, which can incorporate both metadata information and word embeddings within a topic model.
 Since the field of incorporating word embeddings within generative topic models have gained popularity\cite{dieng2020topic}, \citet{MetaLDA2017} show how to use this information for a variety of different datasets.
 They also compare against a list of other models that take either metadata information or word embeddings into account when doing inference.
+We take inspiration from \citet{MetaLDA2017} but we do not employ the model they present.
+We use the Author-Topic, presented by \citet{author_topic_2012}. 
+We use this model instead of the MetaLDA because each document has a specific Dirichlet prior for its topic distribution, which makes it hard to analyze a model that includes multiple metadata fields.
```

Comment:
```suggestion
We use this model instead of MetaLDA because, in MetaLDA, each document has a specific Dirichlet prior for its topic distribution, making it difficult to analyze a model that includes multiple metadata fields.
```